src/authentication/authentication.controller.ts

```diff
@@ -37,4 +52,66 @@ export class AuthenticationController {
       }),
     }
   }
+
+  @Post('send-recover-password-email')
+  @HttpCode(HttpStatus.NO_CONTENT)
+  async sendRecoverPasswordEmail(@Body() { email }: RecoverPasswordSchema) {
+    let user = await this.usersService.findUserByEmail(email)
+
+    if (!user) {
+      // don't want to expose that user with email doesn't exist
+      return
+    }
+
+    // generate and set reset token on user
+    const token = this.authenticationService.generatePasswordResetToken()
+    user = await this.usersService.setResetToken(user.id, token)
+    return await this.emailService.sendRecoverPasswordEmail(user)
+  }
+
+  @Roles(UserRole.admin)
```

Comment:
I think this one needs the `UserRole.sales` role to have access as well. `canCreateCampaigns` here is handling admins' and sales users' access: https://github.com/thegoodparty/tgp-api/blob/62b9ed816d49628efb3dde221479d82556befcd5/config/policies.js#L36

```suggestion
  @Roles(UserRole.admin, UserRole.sales)
```